includes/html/pages/device/port/xdsl.inc.php

```diff
@@ -26,3 +26,16 @@
 
     include 'includes/html/print-interface-graphs.inc.php';
 }
+
+if (file_exists(get_port_rrdfile_path($device['hostname'], $port['port_id'], 'xdsl2LineStatusAttainableRate'))) {
```

Comment:
You cannot use file_exists() for rrd files, you must use Rrd::checkRrdExists().  file_exists() doesn't work with rrdcached.

Reply:
will do.